docs/src/explanations/connectable.md

```diff
@@ -617,7 +644,7 @@ class Example13 extends RawModule {
 }
 ```
 
-Note that the `bits` fields ARE connected, even though they are waived, as `waive` just changes whether an error should be thrown if they are missing, NOT to not connect them if they are structurally equivalent.
+Note that the `bits` fields ARE connected, even though they are waived, as `waive` just changes whether an error should be thrown if they are missing, NOT to not connect them if they are structurally equivalent. To always emit the connection, use `exclude` instead of `waive`.
```

Comment:
```suggestion
Note that the `bits` fields ARE connected, even though they are waived, as `waive` just changes whether an error should be thrown if they are missing, NOT to not connect them if they are structurally equivalent. To always omit the connection, use `exclude` instead of `waive`.
```

Another place you might want to clarify, if there is just an `exclude` and they mismatch, is that an error? So if you are saying "I'm not sure if this is here on both sides, no matter what, don't connect them" do I have to `exclude` AND `waive`?